web/styles.css

```diff
@@ -13,6 +12,7 @@ body {
 }
 
 #app {
+    overflow: hidden;
```

Comment:
Is this needed? Remove if not, prevents edge cases where we accidentally trim content, so users can at least see it

Reply:
this fixes the issue you noticed earlier with things breaking when you are changing the screensize inside of devtools